lib/ex_sdp/utils.ex

```diff
@@ -63,4 +63,16 @@ defmodule ExSDP.Utils do
       {:error, :invalid_sprop_parameter_sets}
     end
   end
+
+  @spec parse_sprop_ps(binary()) :: {:error, :invalid_sprop_parameter_sets} | {:ok, [binary()]}
```

Comment:
Since H265 does not have a field explicitly named `sprop_parameter_sets`, I think we should change this error.

I see two possibilities:
1. we make both this function and the one above it return something like `{:error, :invalid_parameter_sets}`
2. we change the error here to `{:error, :invalid_ps}` and leave the H264 one as-is

@mickel8 what do you think?

Reply:
`:invalid_ps` sounds ok